### ISSUE TITLE
Glitchy warping/cloning with pAI enabled medbot and tractor

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#17785]
(WEB REPORT BY: spooooon REMOTE: 172.93.109.202:7777)
# Revision
266e6c598b6b5f9d6ed332166957b4d194fa448f

# Description
In a similar vein to the ED209/mulebot pAI bug report, this one involves the tractor carts and a pAI enabled medbot.  If I get on and off a tractor cart as a pAImedbot it will leave a 'clone' of me on the tractor.  This will either stay until it is far enough away to rubberband me to the new location, or if I click on it to get on it, make the 'clone' disappear.  
# Steps to Reproduce

# What you Expected

# What Actually Happened

**Comment by Kurfursten** [comment]
@Exxion I do not believe you can even get on a tractor cart anymore as a pAI as of the tractor changes. I couldn't get on in testing.
